### ISSUE TITLE
updated 0.3.8 to 0.3.9 in hassio 2022.2.9 and it stopped working

### ISSUE THREAD

**Issue by RenfoJu1** [JoDehli/PyLoxone#107]
This error originated from a custom integration.

Logger: homeassistant.config_entries
Source: custom_components/loxone/api.py:84
Integration: PyLoxone (documentation, issues)
First occurred: 2:03:13 PM (1 occurrences)
Last logged: 2:03:13 PM

Error setting up entry PyLoxone for loxone
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 327, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/loxone/__init__.py", line 141, in async_setup_entry
    if not await miniserver.async_setup():
  File "/config/custom_components/loxone/miniserver.py", line 118, in async_setup
    request_code = await self.lox_config.getJson()
  File "/config/custom_components/loxone/api.py", line 121, in getJson
    api_resp = await client.get("/jdev/cfg/apiKey")
  File "/usr/local/lib/python3.9/site-packages/httpx/_client.py", line 1736, in get
    return await self.request(
  File "/usr/local/lib/python3.9/site-packages/httpx/_client.py", line 1513, in request
    return await self.send(request, auth=auth, follow_redirects=follow_redirects)
  File "/usr/local/lib/python3.9/site-packages/httpx/_client.py", line 1600, in send
    response = await self._send_handling_auth(
  File "/usr/local/lib/python3.9/site-packages/httpx/_client.py", line 1628, in _send_handling_auth
    response = await self._send_handling_redirects(
  File "/usr/local/lib/python3.9/site-packages/httpx/_client.py", line 1686, in _send_handling_redirects
    raise exc
  File "/usr/local/lib/python3.9/site-packages/httpx/_client.py", line 1668, in _send_handling_redirects
    await hook(response)
  File "/config/custom_components/loxone/api.py", line 84, in raise_if_not_200
    raise LoxoneHTTPStatusError(
custom_components.loxone.api.LoxoneHTTPStatusError: Miniserver response code 307

**Comment by JoDehli** [comment]
Update to the newest home assistant version.

**Comment by JoDehli** [comment]
or downgrade to 0.3.8 they removed in the newest version of home assistant a package which was required in 0.3.8. So if you use 0.3.9 with a old version of home assistant you get this error.

**Comment by JoDehli** [comment]
I you use a external url go back to 0.3.8 because this is not supported at the moment.

**Comment by RenfoJu1** [comment]
Thanks!
got it working with internal address.

**Comment by YannickG1** [comment]
@JoDehli ,

I had the same issue and indeed using local ip is working. I'm just wondering is there any difference in functionality using local ip instead of dns address since your readme states that for gen2 you need to use dns  address ? Is it just to be able to use ssl 443 ? (since ssl certificates cannot bind to ip address)

**Comment by JoDehli** [comment]
I think for gen1 it makes no difference for the security.

**Comment by YannickG1** [comment]
> I think for gen1 it makes no difference for the security.

I'm using gen2 though not gen1

**Comment by JoDehli** [comment]
Have you tried the master branch? I tried to fix it but I have no possibility to test it.

**Comment by JoDehli** [comment]
@YannickG1 have you tried it or is it not important?

**Comment by YannickG1** [comment]
I’m using hacs to install so I don’t think I can easily test the master branch ? In any case everything seems to be working fine using the ip address so for me it is even better like this because no dns resolution can fail. You might just need to update your readme to state that also for gen 2 you can use ip address, no need to use loxone dns.

**Comment by JoDehli** [comment]
Ok with hacs it should be really easy. You can go on reinstall an choose the master branch.

For a other user it worked with the following settings in loxone:

![image](https://user-images.githubusercontent.com/32484282/157948136-082dad77-079b-4197-8eed-267af6efb3da.png)

![image](https://user-images.githubusercontent.com/32484282/157947982-3f84edc4-1cbf-490d-82fc-09a8fb60d479.png)

Maybe you have something different there.

**Comment by YannickG1** [comment]
Hello @JoDehli ,

I have tested it for you. Difference between above screenshot and my config was the checkbox to allow Loxone to be remotely configurable was disabled on my config. I have enabled it and switched to your master branch instead of 0.3.9 but it's still the exact same error :)  I am switching off loxone remote access now and revert to local ip and 0.3.9 which is working fine with latest HA version.
